server/webhook_worker.go

```diff
@@ -56,16 +56,13 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err != nil {
 		return err
 	}
-
-	if _, _, err = wh.PostNotifications(ww.p, msg.InstanceID); err != nil {
```

Comment:
It looks like we are no longer posting the notifications here

Reply:
Sorry my mistake it got removed while cleaning code